tests/DummyData.spec.js

```diff
@@ -87,28 +93,39 @@ Dummy.accountStore = new AccountStore(Dummy.DUMMY_ACCOUNT_DATABASE_NAME);
         db.close();
     }
 
+    async function createDummyKeyStore() {
+        // The key store can be created and filled by its api
+        await KeyStore.instance.connect();
+        await Promise.all([
+            KeyStore.instance.putPlain(Dummy.keyDatabaseEntries[0]),
+            KeyStore.instance.putPlain(Dummy.keyDatabaseEntries[1])
+        ]);
+        await KeyStore.instance.close();
+    }
+
     /** @param {string} dbName */
     async function deleteDatabase(dbName) {
         return new Promise((resolve, reject) => {
             const request = indexedDB.deleteDatabase(dbName);
             request.onerror = () => reject;
             request.onsuccess = resolve;
+            request.onblocked = () => reject(new Error('Can\'t delete database, there is still an open connection.'));
         });
     }
 
     beforeAll(async () => {
         Nimiq.GenesisConfig.test();
         await Promise.all([
             Nimiq.WasmHelper.doImportBrowser(),
-            createDummyAccountStore()
+            createDummyAccountStore(),
+            createDummyKeyStore()
```

Comment:
I think the creation and dropping of the database should be done `beforeEach`/`afterEach` test to have a clean slate. Jasmine executes the tests of the same `describe` after each other.

Reply:
Well, it would be slower. I make sure myself that all tests return the database to the initial state, if they changed something which only applies to add and remove tests

Reply:
Best practice is to always create and tear-down the environment for each test. That way no test is infuenced by other tests, they are independent.